Docs: clarify namespace groupName escaping in the rule config API path segments (#8969)
grafana/mimir: docs/sources/mimir/references/http-api/index.md

```

Returns the rule groups defined for a given namespace.

> The `{namespace}` path segment should be escaped using percent-encoding as defined by

Reviewer: ```suggestion
Escape the `{namespace}` path segment using percent-encoding, as defined by
```

> [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) (for example, `/` should escaped to `%2F`).

Reviewer: ```suggestion
[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986). For example, escape `/` to `%2F`.
```

```

Returns the rule group matching the request namespace and group name.

> The `{namespace}` and `{groupName}` path segments should be escaped using percent-encoding as defined by
> [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986) (for example, `/` should escaped to `%2F`).

Reviewer: ```suggestion
Escape the `{namespace}` and `{groupName}` path segments using percent-encoding, as defined by
[RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986). For example, escape `/` to `%2F`.
```

Creates or updates a rule group.
This endpoint expects a request with `Content-Type: application/yaml` header and the rules group **YAML** definition in the request body, and returns `202` on success.
The request body must contain the definition of one and only one rule group.

> The `{namespace}` path segment should be escaped using percent-encoding as defined by [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986)
> (for example, `/` should escaped to `%2F`).

Reviewer: ```suggestion
Escape the `{namespace}` path segment using percent-encoding, as defined by [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986).
For example, escape `/` to `%2F`.
```

```

Deletes a rule group by namespace and group name. This endpoints returns `202` on success.

> The `{namespace}` and `{groupName}` path segments should be escaped using percent-encoding as defined by [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986)
> (for example, `/` should escaped to `%2F`).

Reviewer: ```suggestion
Escape the `{namespace}` and `{groupName}` path segments using percent-encoding, as defined by [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986).
For example, escape `/` to `%2F`.
```

```

Deletes all the rule groups in a namespace (including the namespace itself). This endpoint returns `202` on success.

> The `{namespace}` path segment should be escaped using percent-encoding as defined by [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986)
> (for example, `/` should escaped to `%2F`).

Reviewer: ```suggestion
Escape the `{namespace}` path segment using percent-encoding, as defined by [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986).
For example, escape `/` to `%2F`.
```
